### PR TITLE
HOMS-173 Add Show More button for orders list

### PR DESCRIPTION
**Pull request by sugarfree1**

### CODE REVIEW

File: app/assets/javascripts/components/order_list.jsx [hunk 1]
```diff
@@ -69,19 +72,47 @@ var OrderList = React.createClass({
...
+  },
+
   render() {
     var options = [];
     for (var field in this.props.profile.data) {
       var disabled = field == 'code';
       options.push(<option key={field} value={field} disabled={disabled}>{this.props.profile.data[field].label}</option>)
     }
 
+    var shorMoreContainer;
```
Comment (TimAle) [inline app/assets/javascripts/components/order_list.jsx]: Typo.

File: app/assets/stylesheets/framework_and_overrides.sass [hunk 2]
```diff
@@ -74,8 +77,18 @@ main.container
...
+
     .order-list-table
       overflow: auto
+      margin-bottom: 21px
+
+    .show-more-container
+      text-align: center
+
+    .show-more-container:hover
+      background-color: #f5f5f5
```
Comment (TimAle) [inline app/assets/stylesheets/framework_and_overrides.sass]: How about `cursor: pointer`?

**Comment by TimAle** [comment]
Add ajax loading indicator.

**Comment by sugarfree1** [comment]
Updated

**Comment by sugarfree1** [comment]
Tests are green.

**Pull request merged**